core/dynlib/lib_windows.odin

```diff
@@ -1,3 +1,4 @@
+// +build windows
```

Comment:
This is not needed if the file name has `_windows.odin`, however it does hurt to have it.